### ISSUE TITLE
No showing dimensions when you resizing a selection

### ISSUE THREAD

**Issue by realOgenki** [lupoDharkael/flameshot#595]
**Flameshot version**
0.6.0-3

**Request**
When you are resizing the selection with the pointer the dimension are not shown until you release the button of your mouse. I expect when dimensions button is pushed you can see the dimensions while you are resizing to choose the size that you want.

**System Information**
Arch Linux (Anarchy installer) with 5.2.9-arch1-1-ARCH kernel.

**Comment by a-tokarev** [comment]
YES! Would be a very useful feature (present in macOS Screenshot utility, by the way), because it would allow pixel-perfect, **exact** screenshots on the go, compared to guessing or post-resizing.
Thus, **Feature Request**: Display Selection Dimensions (e.g., 824x643, etc) While Dragging/Resizing

**Comment by apcrcom** [comment]
Please. Have used other tools with this feature and its very handy for taking quick px measurements of screen elements.

**Comment by mmahmoudian** [comment]
It has been discussed with @borgmanJeremy [on Slack](https://flameshotworkspace.slack.com/archives/C019W6EPYMN/p1600443991007300)

**Comment by t-lock** [comment]
:+1:  Plus one

**Comment by eamodio** [comment]
I would love this as well, and ideally outside the bubble style -- more of a 500 x 120 indicator anchored to the top left of the selection

**Comment by brizzbane** [comment]
I'm newb to this, but I googled because I can't even get dimensions to show at any point.

**Comment by RyonMB** [comment]
@brizzbane I have the same
